CMakeLists.txt

```diff
@@ -12,12 +12,20 @@ option(BUILD_SHARED_LIBS "Build shared libraries." OFF)
 option(BUILD_PCAP "Build pcap utils." OFF)
 option(BUILD_TESTING "Build tests" OFF)
 option(BUILD_VIZ "Build Ouster visualizer." ON)
+option(BUILD_EXAMPLES "Build C++ examples" OFF)
+option(EIGEN_MAX_ALIGN_BYTES "Eigen max aligned bytes." 16)
```

Comment:
Would you mind changing this to `USE_EIGEN_MAX_ALIGN_BYTES` to avoid collisions? 

Also, we would prefer to keep the default 32!

Reply:
sure, as u like